### PR TITLE
Add build instructions for audio sensors

### PR DESCRIPTION
**Pull request by sanchitgarg**
## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: BUILD_FROM_SOURCE.md [hunk 1]
```diff
@@ -86,6 +86,13 @@ We highly recommend installing a [miniconda](https://docs.conda.io/en/latest/min
    python setup.py install --bullet    # build habitat with bullet physics
    ```
 
+   With audio sensor via [rlr-audio-propagation](https://github.com/facebookresearch/rlr-audio-propagation/):
+   To use Audio sensors, enable the audio flag via:
+
+   ```bash
+   python setup.py install --audio    # build habitat with audio sensor
```
Comment (dhruvbatra) [inline BUILD_FROM_SOURCE.md:93]: Nice :-)

**Review by aclegg3 — APPROVED** [review]
LGTM, thanks for adding docs. :)

**Comment by aclegg3** [comment]
One thought, we may want to mention this feature support Linux only at this time.

**Review by Skylion007 — APPROVED** [review]
Agree with Alex on saying it only supports Linux.

**Pull request merged**